pyproject.toml

```diff
@@ -17,48 +17,25 @@ pyyaml = "*"
 
 [tool.poetry.group.dev.dependencies]
 bandit = "*"
-black = "*"
-coverage = "^7.3.2"
+coverage = "*"
```

Comment:
I think we need to pin coverage for the same reason we pinned it in core:
https://github.com/nautobot/nautobot/blob/8dbbdaa307c90a5e315257c1d4f866c5b0f263af/pyproject.toml#L217

Newer versions cause problems with pdb breakpoints

Reply:
Will follow up later https://github.com/nautobot/cookiecutter-nautobot-app/issues/136